src/main/java/frc/robot/subsystems/gripper/Commands/toggleSolenoid.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.subsystems.gripper.Commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.subsystems.gripper.Gripper;
+
+public class toggleSolenoid extends CommandBase {
+    private Gripper gripper;
```

Comment:
Please initialize the gripper in the constructor and `addRequirements`

Reply:
done.